source/upgrade.txt

```diff
@@ -61,32 +61,27 @@ the version after v4.0 including any listed under v4.5.
 Version 5.0 Breaking Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- This driver version introduces the following changes to the ``ConnectionId`` class:
-  
-  - The ``ConnectionId`` constructor now accepts a value of type ``long`` as its second
-    parameter instead of type ``int``. Similarly, the constructor now accepts a value of
-    type ``Long`` as its third parameter instead of type ``Integer``. Because this change breaks
-    binary compatibility, recompile any existing code that calls the ``ConnectionId`` constructor.
-
-  - The ``withServerValue()`` method now accepts a parameter of type ``long`` rather than
-    type ``int``. This change breaks binary compatibility, so you must recompile any code
-    that calls the ``withServerValue()`` method.
-
-  - The ``getServerValue()`` method now returns a value of type ``Long`` instead of type
-    ``Integer``. Similarly, the ``getLocalValue()`` method returns a value of type
-    ``long`` instead of type ``int``. Because this change breaks both binary and source
-    compatibility, update any source code that uses these methods and rebuild your binary.
+This driver version introduces the following breaking changes:
+
+- Changes the data type of the timeout duration parameter in the
+  ``SocketSettings.Builder.connectTimeout()`` and ``SocketSettings.Builder.readTimeout()``
+  methods. The data type of this parameter is now ``long`` instead of ``int``.
+  To view an
+  example that shows how to instantiate a ``SocketSettings`` instance by using
+  these methods, see the :ref:`SocketSettings Example
+  <java-socketsettings-example>` in the Specify MongoClient Settings
+  guide.
```

Comment:
This text effectively duplicates the text below (search for `connectTimeout`, `readTimeout`, java-socketsettings-example). The latter was reviewed in https://github.com/mongodb/docs-java/pull/507.

```suggestion
```

Reply:
Sorry, I'm not seeing the duplicate text; could you specify where it is?

Reply:
Apologies, that text wasn't duplicated, it was removed and replaced with the text I quoted originally. The removed text has already been reviewed and updated in https://github.com/mongodb/docs-java/pull/507, more specifically, in the following commit: https://github.com/mongodb/docs-java/pull/507/commits/7e55ff2bb5500d7cd4fd855d298f5280810a170f.

The fact that this PR does not have those modifications, yet also updates that text suggests that something went wrong. Anyway, I am asking to incorporate the changes made in the commit https://github.com/mongodb/docs-java/pull/507/commits/7e55ff2bb5500d7cd4fd855d298f5280810a170f to files `source/includes/fundamentals/code-snippets/mcs.java` and `source/upgrade.txt` into the current PR (the list below gives an overview of the changes I am talking about, the linked commit contains all of these changes):

1. The `L` suffixes in the example are not needed.
2. The example does not show how to instantiate `SocketSettings`.
3. We need to inform users that this change breaks binary compatibility.